packages/opentelemetry-api/test/api/api.test.ts

```diff
@@ -34,8 +34,8 @@ describe('API', () => {
 
   describe('GlobalTracerProvider', () => {
     const spanContext = {
-      traceId: 'd4cda95b652f4a1592b449d5929fda1b',
-      spanId: '6e0c63257de34c92',
+      traceId: new Uint8Array([0xd4, 0xcd, 0xa9, 0x5b, 0x65, 0x2f, 0x4a, 0x15, 0x92, 0xb4, 0x49, 0xd5, 0x92, 0x9f, 0xda, 0x1b]),
```

Comment:
For the purpose of readable tests, would it be better to do `Uint8Array.from(Buffer.from('d4cda95b652f4a1592b449d5929fda1b', 'hex'))`?

Reply:
Does this work for Browser?

Reply:
ah I guess not because no buffer